cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TABLE cuws.verified_yield_contract_commodity(
+    verified_yield_contract_commodity_guid    varchar(32)       NOT NULL,
+    verified_yield_contract_guid              varchar(32)       NOT NULL,
+    crop_commodity_id                         numeric(9, 0)     NOT NULL,
+    harvested_acres                           numeric(14, 4),
+    harvested_acres_override                  numeric(14, 4),
+    stored_yield_default_unit                 numeric(14, 4),
+    sold_yield_default_unit                   numeric(14, 4),
+    production_guarantee                      numeric(14, 4),
```

Comment:
Are we sure about storing production_guarantee? Does that mean we'll need to add it to the data sync? I had been thinking we might just load it dynamically whenever the Verified Yield screen is displayed.

Reply:
I would like to use the data sync if we can and move away from accessing CIRRAS at runtime.
I think we should add it to the data sync considering that the PY and deductible level has to be brought over as well. PY is actually used to calculate some values in the yield summary table.

Reply:
Ok, sounds good.